### PR TITLE
Doc/tm4j-6802/multiple testsuite

### PR DESCRIPTION
**Pull request by vmrvictor**
To generate files with multiple test suites at the same time, Zephyr Scale supports the following structure:
```
<testsuites>
    <testsuite>    
    ....    
    </testsuite>
</testsuites>
```

There is already a Robot Framework's [XUnit Output Modifier (xom) ](https://github.com/rikerfi/robotframework-xunitmodifier)

To use it, one of the options is to copy the file xom.py in the directory you are running your scripts.

Edit the content of xom.py file by changing `ROOT_NODE_PLURAL` to:```ROOT_NODE_PLURAL = True```

and then generate the files by:

```robot --pythonpath . --prerebotmodifier xom.XUnitOut:xunit.xml tests```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,8 +97,8 @@ In order to execute this example on your local machine you will have to checkout
 
 ```
 brew upgrade pyenv
-pyenv install 3.6.0
-pyenv global 3.6.0
+pyenv install 3.8.13
```
Comment (vmrvictor) [inline README.md:100]: I have updated python version to be used because on M1 3.6.0 needs additional patches (not supported directly)

File: README.md [hunk 2]
```diff
@@ -16,8 +16,44 @@ robot -x junitresult.xml mytest.robot
...
+brew install wget
+wget https://github.com/rikerfi/robotframework-xunitmodifier/raw/main/xom.py
+```
+Edit the content of xom.py file by changing `ROOT_NODE_PLURAL` to:
+```
+ROOT_NODE_PLURAL = True
+```
+and if you want to run all robot frameworks inside your folder tests then you can run the following command:
+```
+robot --pythonpath . --prerebotmodifier xom.XUnitOut:multipleTestSuiteInTestSuitesUnit.xml tests
```
Comment (vmrvictor) [inline README.md:54]: I have created tests folder to contain different robot scripts, so we can run multiple testsuites at once

File: tests/calculator.robot [hunk 3]
```diff
@@ -0,0 +1,56 @@
+
```
Comment (vmrvictor) [inline tests/calculator.robot:1]: just duplicated from calculator.robot

File: tests/calculator2.robot [hunk 4]
```diff
@@ -0,0 +1,56 @@
+
```
Comment (vmrvictor) [inline tests/calculator2.robot:1]: just duplicated from calculator.robot

File: tests/calculator3.robot [hunk 5]
```diff
@@ -0,0 +1,56 @@
+
```
Comment (vmrvictor) [inline tests/calculator3.robot:1]: just duplicated from calculator.robot

File: README.md [hunk 6]
```diff
@@ -16,8 +16,44 @@ robot -x junitresult.xml mytest.robot
 
 The command line above will execute the `mytest.robot` test file and generate the JUnit XML results file `junitresult.xml`. Then, this file containing the test results can be uploaded to Zephyr Scale using the following API endpoint: [`POST /automations/executions/junit`](https://support.smartbear.com/zephyr-scale-cloud/api-docs/#operation/createJUnitExecutions).
 
-The abovementioned API accepts either a single XML file as well as a .zip file containing multiple XML files. The POST request will create a new test cycle in Zephyr Scale containing the results and will respond with the key of the created test cycle.
+The above mentioned API accepts either a single XML file as well as a .zip file containing multiple XML files. The POST request will create a new test cycle in Zephyr Scale containing the results and will respond with the key of the created test cycle.
 
+### One file containing multiple test-suites
+
+Zephyr Scale does support having more than one testsuite in the same xml file, but RobotFramework is generating them
```
Comment (assisblf) [inline README.md:23]: Super minor and probably personal preference, but I'd rephrase it to:
> Zephyr Scale supports XML files with multiple `testsuite` tags. However, some versions of RobotFramework generate them in a way Zephyr Scale does not recognize, as in this example:

File: README.md [hunk 7]
```diff
@@ -16,8 +16,44 @@ robot -x junitresult.xml mytest.robot
...
+To use it, one of the options is to download the file xom.py in the directory you are running your scripts:
+```
+brew install wget
+wget https://github.com/rikerfi/robotframework-xunitmodifier/raw/main/xom.py
+```
+Edit the content of xom.py file by changing `ROOT_NODE_PLURAL` to:
+```
+ROOT_NODE_PLURAL = True
+```
+and if you want to run all robot frameworks inside your folder tests then you can run the following command:
```
Comment (assisblf) [inline README.md:52]: I'd personally change this to the following text.

> and if you want to run all Robot Framework tests inside a folder called `tests`, you can run the following command:

**Review by assisblf — APPROVED** [review]
Looks good to me! I've just added a few comments, but I guess they are partially just personal preference
